app/controllers/callbacks_controller.rb

```diff
@@ -0,0 +1,7 @@
+class CallbacksController < Devise::OmniauthCallbacksController
+  def github
+    user = User.from_omniauth(request.env["omniauth.auth"])    
+    sign_in_and_redirect user
+    flash[:notice] = "Signed in successfully."
```

Comment:
Please use the translation key, if it's not available on config/locales/en.yml feel free to add it as sign_in_message key
```suggestion
    flash[:notice] = t(:sign_in_message)
```